Create 2019-08-13-Introducing....md

GeorgiaHCA opened the pull request:
Adding this again so people can review it!

Commit:
Create 2018-08-13-Introducing....md

Adding this again so people can review it!

sgibson91 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+
+For her PhD in Astrophysics, Sarah worked on a NASA (:heart_eyes:!!) project called Swift hunting down gamma ray bursts. Gamma ray bursts are very brief flashes of high energy radiation, and it’s difficult to predict when and where they will go off. 
+
+Gamma ray bursts form in two ways: when young massive stars go supernova, or two neutron stars merge together. The first produces gamma ray bursts which are long in time duration because they have a lot of matter. They happen in very early galaxies and are very bright, and studying them tells us a lot about the first galaxies that were formed after the Big Bang. Many galaxies may not be bright en ...
```
```suggestion
Gamma ray bursts form in two ways: when young massive stars go supernova, or two neutron stars merge together. The first produces gamma ray bursts which are long in time duration because they have a lot of matter. They happen in very early galaxies and are very bright, and studying them tells us a lot about the first galaxies that were formed after the Big Bang. Many galaxies may not be bright enough to see without these gamma rays, so studying them can help us discover new galaxies, as well as telling us the composition of stuff between us and the burst - e.g. what our Galaxy is made of, what the galaxy the burst happened in is made of, and what the stuff in between is made of.
```
Very tiny, nit-picky point. The convention is to capitalise "galaxy" when referring to the one we live in 😊

sgibson91 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+
+Dead stars that merge are much older:  you need to have time for two stars to die and then spiral into one another. Studying the gamma rays which neutron stars emit can tell us how often we can expect stars to die and how often collisions occur when stars are orbiting each other.
+
+NASA uses an autonomous satellite which can react rapidly and make decisions without a controller, and Sarah used Monte Carlo methods to optimise her model to the data. She also spent a month in Namibia where there’s an on-ground telescope, looking at gamma rays emitted from black holes at the centre of our galaxy. We can use this information to learn more about how standard the Milky Way is comp ...
```
```suggestion
NASA uses an autonomous satellite which can react rapidly and make decisions without a controller, and Sarah used Monte Carlo methods to optimise her model to the data collected by this satellite. She also spent a month in Namibia where there’s an on-ground telescope, looking at gamma rays emitted from black holes at the centre of our Galaxy. We can use this information to learn more about how standard the Milky Way is compared to other galaxies.
```

sgibson91 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+
+Sarah’s official job title at the Turing is “Research Data Scientist”. There are two distinct roles, data scientist and software engineer, which are often seen as a binary, but the Turing treats them as a spectrum, and Sarah floats between the two. 
+
+As a data scientist, her job is analytical, and involves things like cleaning data sets, chosing the right methods to answer particular research questions, and creating data visualisations. As a software engineer, her work involves building tools, and following best practices when using code to answer questions. She says that it’s very important to remember that code is not infallible, a human ha ...
```
```suggestion
As a data scientist, her job is analytical, and involves things like cleaning data sets, choosing the right methods to answer particular research questions, and creating data visualisations. As a software engineer, her work involves building tools, and following best practices when using code to answer questions. She says that it’s very important to remember that code is not infallible, a human has to create it and we are subject to our own biases and opinions that might be transferred.
```
Typo: "chosing" --> "choosing"

sgibson91 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+
+If you don’t follow best practice, you run the risk of creating unreproducible code, or code that doesn’t do what you think it’s doing. There is the risk of the code producing an answer or output that that isn’t obviously wrong but is still wrong, and this is a real issue if there’s no transparency and no-one can interrogate it.
+
+Sarah is involved in creating The Turing Way. This was originally a guide for doing reproducible data science; but it is evolving into a more general guide. She has been linking up with the Binder community, as they have a great tool for sharing code and data very easily. Binder is a solution to being able to test whether you can get the same results from the same data: this could be by using myb ...
```
```suggestion
Sarah is involved in creating [_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way). This was originally a guide for doing reproducible data science; but it is evolving into a more general guide. She has been linking up with the [Binder](https://mybinder.readthedocs.io/en/latest/) community, as they have a great tool for sharing code and data very easily. Binder is a solution to being able to test whether you can get the same results from the same data: this could be by using [mybinder.org](https://mybinder.org) which is completely public, or by installing your own [BinderHub](https://binderhub.readthedocs.io/en/latest/) and enabling it to work with a private repository. 
```
I thought it might be nice to add some links here to the various projects :)

GeorgiaHCA replied:
Fab- thanks Sarah! Def helpful. :)

sgibson91 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+
+#### “What’s your ideal outcome of your time at the Turing?”
+
+Sarah finds that the Turing gives her the opportunity to work in a lot of different domains and projects. Her ideal outcome would be that through the Turing she can “find my groove” as well as the domain that she is best at and appeals to her most. She’s proud to work with the Binder community and has been asked to be a long-term contributor - anything she could do to contribute to the community  ...
```
```suggestion
Sarah finds that the Turing gives her the opportunity to work in a lot of different domains and projects. Her ideal outcome would be that through the Turing she can “find her groove” as well as the domain that she is best at and appeals to her most. She’s proud to work with the Binder community and has been asked to be a long-term contributor - anything she could do to contribute to the community and push technology forward would make her very proud. 
```
I think this sentence reads better if you continue to refer to me in third person ✨

sgibson91 reviewed (approved):
Hi @GeorgiaHCA, thank you for the lovely blog! I've left a few minor comments but will approve 💖

Btw, a Markdown/GitHub hack: I prefer to start each new sentence on a new line when writing Markdown. This is easier to review on GitHub as the whole paragraph doesn't get captured in the suggested changes (as it has in my suggestions below). And Markdown knows to still render all the individual sentences as a single paragraph - it creates a new paragraph when there's a blank new line - so it will still look the same when published.

What you've done is by no means wrong, but you might find this tip useful in the future 😊

minnieho1115 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+#### “Tell me about your background”
+
+Ang is currently a PhD student. His college major was in engineering, specialising in space flight. He studied in the Northeast of China in the Heliongjiang Province at the University of Harbin. Ang was born in a very small city. At first, he didn’t know that much about engineering, but he was told that it was very lucrative! When he started studying, however, he found that he was not so interest ...
+
```
He studied in the Northeast of China in the **_Heilongjiang_** Province **at the Harbin Institute of Technology**. Ang was born in a very small city. At first, he didn’t know that much about engineering, but he was told that it was very lucrative! When he started studying, however, he found that he was not so interested in engineering and switched to brain science. He is very interested in the biology of the brain.

minnieho1115 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+#### “Tell me about your background” 
+
+Yini majored in psychology and is currently doing a PhD in neuroimaging. She changed her subject because during the last year of her undergraduate degree she started to be an intern teacher in a special school, and a lot of students had mental health challenges such as depression, bipolar disorder, and schizophrenia. Yini found her time teaching them an amazing experience. She enjoyed their creat ...
+
```
Yini majored in psychology and is currently doing a PhD in neuroimaging. She changed her subject because during the last year of her undergraduate degree she started to be an intern teacher in a special school, and a lot of students had mental health challenges such as depression, bipolar disorder, and schizophrenia. Yini found her time teaching them an amazing experience. She enjoyed their creativity and talking to them, and she wanted to do things to help them and to change their lives for the better. **She thought she had to know more about neural mechanisms of mental disorders in order to gain a deeper understanding of them**,so she decided to change her field. She started working in a hospital - Xiangya – which is very famous hospital in China – and met people there. She realised she would like to know about adolescents, and to research if it would be possible to screen vulnerability to mental health issues, and maybe use the screening to find ways to make people’s lives better in the future.

minnieho1115 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,96 @@
+#### “How are you finding the Turing?”
+
+“I think my major and interests involve very complex questions, based on a lot of different kinds of knowledge such as neuroimaging, computer programming, and biology, so I would like to communicate with people from different fields, and coming to the Turing I’ve met a lot of amazing people, so it’s very interesting.”
+
```
Making friends with you is  a lucky thing in my life!!!

GeorgiaHCA replied:
Thanks Yini you too :D

minnieho1115 reviewed (commented):
Thank you Georgia!It is a cool interview!!!From your interview, I have a deep understanding of Sara and ang's work.Thank you. You did a great job.My only suggestion is the same as Sara's.Each sentence can be written as a new line:)

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sarah Gibson <44771837+sgibson91@users.noreply.github.com>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sarah Gibson <44771837+sgibson91@users.noreply.github.com>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sarah Gibson <44771837+sgibson91@users.noreply.github.com>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sarah Gibson <44771837+sgibson91@users.noreply.github.com>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sarah Gibson <44771837+sgibson91@users.noreply.github.com>

Commit:
Update 2018-08-13-Introducing....md

Made reviewers suggestions and gave each sentence its own line. Thanks for helping everyone :-)

KirstieJane reviewed (commented):
Hi @GeorgiaHCA!! Thank you for writing this up! Its really interesting - I'm glad you all got to have these conversations!

A couple of formatting things that need to change:

- [x] The file is in the wrong directory - it should be saved in `_posts/blog/2018-08-13-Introducing....md` rather than `_posts/2018-08-13-Introducing....md`
- [x] Having `.....` in the file name is too confusing for jekyll so the site doesn't build by default. I also think that the name needs to be a little clearer for folks reading the blog post list. So maybe "Georgia meets three people at the Turing?" or something like that? I would also love it if you, @minnieho1115 and @sparkler0323 had similar titles so that it was clear that you've all done the same blog post exercise. I'll let you three think of something good.
- [x] Emojis using the `:sparkles:` syntax don't render unfortunately - they need to be copied and pasted from (eg) emojipedia.

![image](https://user-images.githubusercontent.com/3626306/63229241-52655900-c1f6-11e9-8089-088cd298ac90.png)

KirstieJane commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+layout: post
+title: ### Introducing...
+categories: blog
+excerpt: ### blog introducing three people from Kirstie's lab 
```
```suggestion
excerpt: Blog introducing three people from Kirstie's lab 
```

KirstieJane commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: ### Introducing...
```
Title needs to be before the `###` because those are comment symbols so everything after them is ignored! (Actually, you can just delete them - you don't need them there!)

KirstieJane commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+title: ### Introducing...
+categories: blog
+excerpt: ### blog introducing three people from Kirstie's lab 
+tags: [] ### INSERT TAGS IF APPLICABLE
```
```suggestion
tags: [turing-institute, onboarding]
```

KirstieJane commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+author: GeorgiaHCA ###
+---
+
+:tada: :star2: :rocket: Introducing three of my stellar Turing colleagues... 
```
I feel like this needs to say a little bit about why you're doing this blog post? Fine if you'd rather just jump in though. Let me know what you think.

GeorgiaHCA replied:
Yeah I think you're right! It's a bit alarming how quickly it dives in - added a small extra introduction to the introductions :)

KirstieJane commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+
+### <em> Fun Extra Fact </em> 
+
+<em> I found out that Yini and I share a nickname – her friends call her the same Chinese name I was given when I worked in Beijing – Hwa Hwa – which means flower! So we have a special connection. </em> :cherry_blossom::cherry_blossom:
```
This is so cooooool!

🌸🌸🌸

KirstieJane reviewed (commented):
Sorry for leaving the review in two parts - I clicked enter too early by accident!

Great blog - well done!

sparkler0323 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+
+Ang would like to learn about reproducible research from Kirstie. His area normally deals with many different types of data, for instance genome data, gene expression and MRI (magnetic resonance imaging) data. 
+As the real real-world questions, especially in brain science, are always complex, expertise from a lot of different domains is required to answer them: psychology, psychiatry, genomic science, mathematics and even physics. 
+Ang believes that a diverse community of people from different specialisms is a necessary to answer some important scientific question. 
```
```suggestion
Ang believes that a diverse community of people from different specialisms is necessary to answer some important scientific questions. 
```

sparkler0323 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+
+#### “Tell me about your research at the Turing”
+
+Ang is interested in mental disorders, especially schizophrenia, and developing clinically useful biomarkers to help psychiatrists by using genomic or neuroimaging data. 
```
```suggestion
Ang is interested in mental disorders, especially schizophrenia, and he is trying to develop clinically useful biomarkers to help psychiatrists based on neuroimaging or genome data. 
```

sparkler0323 commented on _posts/2018-08-13-Introducing....md:
```diff
@@ -0,0 +1,142 @@
+
+Ang is interested in mental disorders, especially schizophrenia, and developing clinically useful biomarkers to help psychiatrists by using genomic or neuroimaging data. 
+Ang would also like to use genomic data to understand the pathology of schizophrenia. 
+The pathology itself is complex, but it is a very important area to understand, and many answers emerge from examining the genome. 
```
```suggestion
The pathology itself is complex, but it is a very important area to understand given there's no known cure, and many answers emerge from examining the genome. 
```

GeorgiaHCA replied:
Sounds like you are doing some amazing work! 
It's not my specialism, but I don't know if everyone would describe it as a disease? What are your thoughts @KirstieJane?

KirstieJane commented:
Hi @GeorgiaHCA - I know you aren't well but I'm away next week so this is just a ping to make sure you don't forget about incorporating the reviews here.

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Kirstie Whitaker <kw401@cam.ac.uk>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Kirstie Whitaker <kw401@cam.ac.uk>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sparkler <ang.li@nlpr.ia.ac.cn>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sparkler <ang.li@nlpr.ia.ac.cn>

Commit:
Update _posts/2018-08-13-Introducing....md

Co-Authored-By: Sparkler <ang.li@nlpr.ia.ac.cn>

Commit:
Rename _posts/2018-08-13-Introducing....md to _posts/blog/2018-08-13-Georgia_Introduces.md

Commit:
Update 2018-08-13-Georgia_Introduces.md

Made changes to introducing... blog post responding to comments.

Commit:
Delete 2019-08-13-Introducing....md

Deleting old file as replaced by new one (Georgia Introduces)

Commit:
Rename 2018-08-13-Georgia_Introduces.md to 2018-08-13-Georgia-Introduces.md

Minor change to title format

KirstieJane commented:
Hi @GeorgiaHCA - I've updated the master branch to fix some broken links - please can you update your branch so that both CIs pass?

KirstieJane commented:
Pinging again - I'd like to get this blog post merged - its been open for 2 months.

Let me know if you have any questions about updating the branch @GeorgiaHCA

KirstieJane committed:
Fix broken links

Commit:
Merge branch 'master' of https://github.com/WhitakerLab/whitakerlab.github.io into GeorgiaHCA-patch-3

GeorgiaHCA commented:
Should be ready to merge - thanks for all your help Kirstie!
